accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/installhook/AcToolInstallHook.java

```diff
@@ -25,7 +26,8 @@
 public class AcToolInstallHook extends OsgiAwareInstallHook {
 
     private static final Logger LOG = LoggerFactory.getLogger(AcToolInstallHook.class);
-    private static final String PROPERTY_ACTOOL_INSTALL_AT_INSTALLED_PHASE = "actool.atinstalledphase";
+    private static final String PROPERTY_ACTOOL_INSTALL_AT_INSTALLED_PHASE = "actool.atInstalledPhase";
```

Comment:
The property didn't work as intended anyway (and we released only one version) - let's use the opportunity to use proper camel case for the property name (for readability)

Reply:
if we really want to do this, the documentation needs to be updated as well

Reply:
yes, docu is already updated